reference/3.0/Microsoft.PowerShell.Utility/Write-Debug.md

```diff
@@ -1,16 +1,16 @@
 ﻿---
-ms.date:  06/09/2017
-schema:  2.0.0
-locale:  en-us
-keywords:  powershell,cmdlet
-online version:  http://go.microsoft.com/fwlink/?LinkID=113424
-external help file:  Microsoft.PowerShell.Commands.Utility.dll-Help.xml
-title:  Write-Debug
+external help file: Microsoft.PowerShell.Commands.Utility.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Utility
+ms.date: 06/09/2017
```

Comment:
Enough content updated to trigger date change.

Reply:
Fixed